api/go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/google/pprof v0.0.0-20230323073829-e72429f035bd // indirect
 	github.com/google/uuid v1.3.0 // indirect
-	github.com/imdario/mergo v0.3.16 // indirect
+	github.com/imdario/mergo v1.0.0 // indirect
```

Comment:
There is no `github.com/imdario/mergo v1.0.0` it's `dario.cat/mergo`. So we can't do this until the module that requires it updates as well:
```
❯ go mod why github.com/imdario/mergo
go: downloading github.com/openstack-k8s-operators/lib-common/modules/database v0.0.0-20230714104628-12cc1e43cccd
# github.com/imdario/mergo
github.com/openstack-k8s-operators/heat-operator
sigs.k8s.io/controller-runtime/pkg/client/config
k8s.io/client-go/tools/clientcmd
github.com/imdario/mergo
```